src/Agent.Plugins/PipelineArtifact/PipelineArtifactPluginV1.cs

```diff
@@ -20,6 +20,8 @@ public abstract class PipelineArtifactTaskPluginBaseV1 : IAgentTaskPlugin
     {
         public abstract Guid Id { get; }
         public virtual string Version => "1.0.0"; // Publish and Download tasks will be always on the same version.
+        public virtual string DownloadPath => "downloadPath";
+        public virtual string BuildId => "buildId";
```

Comment:
wait default it to the correct stuff and have 1_1_0 be the override

Reply:
Ya, was thinking about that. Let me change it.